src/nspawn/nspawn-mount.c

```diff
@@ -711,6 +711,23 @@ int mount_all(const char *dest,
                                 o = options;
                 }
 
+                if (FLAGS_SET(m->mount_settings, MOUNT_TMPFS_USRQUOTA)) {
+                        r = mount_option_supported("tmpfs", /* key= */ "usrquota", /* value= */ NULL);
+                        if (r < 0)
+                                log_warning_errno(r, "Failed to determine if 'tmpfs' supports 'usrquota', assuming it doesn't: %m");
+                        else if (r == 0)
+                                log_warning("Kernel doesn't support 'usrquota' on 'tmpfs', not including in mount options for '%s'.", m->where);
```

Comment:
Should this be `log_info()`, like in mount-setup.c ?

BTW, the logic is almost same as the one added in the first commit.
Maybe, better to introduce a tiny helper for this?